posts/v2-metrics.md

```diff
@@ -0,0 +1,54 @@
+A couple of weeks ago, I posted an issue about our upcoming journey towards Exercism V3. Someone [posted a comment](https://github.com/exercism/exercism/issues/5135#issuecomment-566125777) saying that they hoped the transition would be smoother than V1 to V2, and questioning whether a lot of changes we made were worth the hassle. Both are excellent points to raise and show that we've clearly not explained the impact that the V2 changes **did** have and why we believe they were worth it. I thought it would, therefore, be worth writing a short post outlining the changes we saw in the data, and also the metrics that we're targeting with the push to V3.
+
+Two notes. Firstly, the metrics below are roughly ordered by the flow of the site, rather than of importance. Secondly, I've used a 17-month window on either side of the launch of V2 for comparisons. When I say "V1", I mean the final 17 months of V1. There's no entirely accurate way to do this, as V1 had many small changes in the 17 months prior to launch, and V2 had substantial improvements over the period since it launched. We also have outlier days, like the one during V1 when GitHub featured us, and we had 30x traffic for a day or so. I've not worried about that and just stuck to a fixed window either side, believing it to be the fairest way to measure things.
+
+## V1 -> V2 Metrics
+
+### Visitors signing up (Branding)
+
+One of our aims was to make Exercism feel more welcoming to people who might have been put off by the original (red + devil) branding. To many, this seemed like a minor change, because they **weren't** put off by the branding and so the effort seemed like a waste of time. However, we believed that Exercism didn't exude "welcoming, inclusive community" in V1, and it was important for us to put that right.
+
+How many people signed up to Exercism over the periods, and what was this as a percentage of new visitors?
+
+V1: 62,618 (17% of new visitors sign up)
+V2: 188,844 (48% of new visitors sign up)
+
+This first change was substantial. Our V2 landing page converts at 48% as opposed to 17% for V1. That's pretty much purely down to brand and clarity over what we do and who we are. So I strongly believe that the effort that went into rebranding Exercism was worth the effort.
+
+One caveat is that V2 probably encourages people to sign up who aren't yet ready for Exercism as it stands. V2 isn't set up for total beginners, yet the landing page for V2 is much more welcoming for total beginners, so we probably get a lot of users who aren't yet ready for Exercism. One way to account for this would be to consider not only the number of visitors signing up but rather how many visitors signed up _and_ managed to submit a solution. For this, we see:
+
+V1: 26,194 (7% new visitors sign up and submit)
+V2: 52,533 (13.9% new visitors sign up and submit)
```

Comment:
All of the V1 to V2 comparisons will show up as a single paragraph in the blog post, not as separate lines. Is that intentional?

Reply:
No. I'll fix. Thank you!